src/components/download/helperFunctions.js

```diff
@@ -249,14 +249,15 @@ export const strainTSV = (dispatch, filePrefix, nodes, nodeVisibilities) => {
     const nodeAttrsToIgnore = ["author", "div", "num_date", "vaccine", "accession"];
     const traits = Object.keys(node.node_attrs).filter((k) => !nodeAttrsToIgnore.includes(k));
     for (const trait of traits) {
-      if (!headerFields.includes(trait)) headerFields.push(trait);
       const value = getTraitFromNode(node, trait);
       if (value) {
+        headerInsert(headerFields, null, trait);
```

Comment:
_completely separate tangent_

I just realized this means any `0` value will not be included in metadata downloads. Clearly it hasn't been an issue since I don't see any related bug reports, but probably would be good to fix:

```suggestion
      if (value !== undefined) {
        headerInsert(headerFields, null, trait);
```

Reply:
Good find! Fixed